redis/asyncio/client.py

```diff
@@ -294,6 +294,13 @@ def __init__(
                 "lib_version": lib_version,
                 "redis_connect_func": redis_connect_func,
                 "protocol": protocol,
+                "cache_enable": cache_enable,
```

Comment:
Let's have you, @vladvildanov and @uglide sync on these just to finalize before the **next** release. Getting them into the specs finally too - WDYT?